### ISSUE TITLE
Reproduction problem of the nuscenes dataset

### ISSUE THREAD

**Issue by XiaofengLi-HKUST** [weiyithu/SurroundDepth#10]
Hello, Your results are great, I'm following your work.
I have a problem of reproducing the results for the nuScenes dataset, the abs_rel by Monodepth2 for the nuScenes dataset are high to 0.35, and the curve during training is very strange. 
![1665560657308](https://user-images.githubusercontent.com/115617528/195281640-3fd27698-6046-4ffa-8cda-12f040b64834.jpg)
I use the code of Monodepth2, and use your code to process the dataset.
![103](https://user-images.githubusercontent.com/115617528/195281856-bb6f79ba-d5bd-4d01-abd4-6e227f5dece7.png)
How can I reproduce the baseline results in your paper？
Thanks！

**Comment by XiaofengLi-HKUST** [comment]
May I ask you about the method for calculating the error in your paper. Do you calculate the error of the whole picture or crop some areas? 
Thanks!

**Comment by weiyithu** [comment]
Hi, in my experiment, I find the models are easy to be overfitted on nuScenes. However, high to 0.35 is still strange. Do you use my nuScenes dataset list? where the static frames are removed.

**Comment by XiaofengLi-HKUST** [comment]
yes, I use your nuScenes data list.

**Comment by weiyithu** [comment]
Except some hyperparameters, I think there does not exist differences. Maybe the only difference is that nuscenes is much easier to overfit.

**Comment by XiaofengLi-HKUST** [comment]
I have another question, I also can't reproduce the results of PackNet model. I find that the results of PackNet model in your paper is very different from the result of his original paper in DDAD dataset,  may I ask whether the result of PackNet in your paper is reproduced by youself? 
Thanks!

**Comment by weiyithu** [comment]
Yeah, it is produced myself since the original paper only report front view results but we need all views' results.
